public/components/datasources/components/manage/accelerations/acceleration_table.tsx

```diff
@@ -0,0 +1,201 @@
+      <>
+        <EuiFlexGroup direction="row" alignItems="center">
+          <EuiFlexItem>
+            <EuiText>
+              <h3 className="panel-title">Accelerations</h3>
+              <p>
+                Accelerations optimize query performance by indexing external data into OpenSearch.
+              </p>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <RefreshButton />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <CreateButton />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </>
+    );
+  };
+
+  const tableActions = [
+    {
+      name: 'Discover',
+      description: 'Open in Discover',
+      icon: 'discoverApp',
+      type: 'icon',
+      onClick: onDiscoverButtonClick,
+    },
+    {
+      name: 'Refresh',
+      description: 'Refresh/Pause/Resume',
+      icon: getRefreshButtonIcon,
+      onClick: onRefreshButtonClick,
+    },
+    {
+      name: 'Delete',
+      description: 'Delete acceleration',
+      icon: 'trash',
+      type: 'icon',
+      onClick: onDeleteButtonClick,
+    },
+  ];
+
+  const columns: Array<EuiBasicTableColumn<any>> = [
+    // TODO: fields should be determined by what the acceleration is
+    // Show N/A if not applicable
+    {
+      field: 'name',
+      name: 'Name',
+      sortable: true,
+      render: (name: string) => <EuiLink onClick={() => setIsFlyoutVisible(true)}>{name}</EuiLink>,
+    },
+    {
+      field: 'status',
+      name: 'Status',
+      sortable: true,
+      render: AccelerationStatus,
+    },
+    {
+      field: 'type',
+      name: 'Type',
+      sortable: true,
+      render: (type: string) => {
+        let label;
+        switch (type) {
+          case 'skip':
+            label = 'Skipping Index';
+            break;
+          case 'mv':
+            label = 'Materialized View';
+            break;
+          case 'ci':
+            label = 'Covering Index';
+            break;
+          default:
+            label = 'AAAAAA';
+        }
+        return <EuiText>{label}</EuiText>;
+      },
+    },
+    {
+      field: 'database',
+      name: 'Database',
+      sortable: true,
+      render: (database: string) => <EuiText>{database}</EuiText>,
+    },
+    {
+      field: 'table',
+      name: 'Table',
+      sortable: true,
+      render: (table: string) => <EuiText>{table}</EuiText>,
+    },
+    {
+      field: 'destination',
+      name: 'Destination Index',
+      sortable: true,
+      render: (destination: string) => (
+        <EuiLink onClick={() => console.log(destination)}>{destination}</EuiLink>
+      ),
+    },
+    {
+      name: 'Actions',
+      actions: tableActions,
+    },
+  ];
+
+  const pagination = {
+    initialPageSize: 10,
+    pageSizeOptions: [10, 20, 50, 100],
+  };
+
+  const sorting = {
+    sort: {
+      field: 'name',
+      direction: 'asc',
+    },
+  };
+
+  return (
+    <>
+      <EuiSpacer />
+      {isFlyoutVisible && (
```

Comment:
Did we try to see if this flyout open's from other plugins? I would recommend to use overlay service as mentioned here: https://github.com/opensearch-project/OpenSearch-Dashboards/tree/790c076fd4c8f38ade5482cede6d92d10792be54/src/plugins/opensearch_dashboards_react#overlays-service

Reply:
+1 im also looking into this.

Reply:
Using the overlay service to render flyouts:
```
const renderAccelerationDetailsFlyout = (acceleration: any) =>
  core.overlays.openFlyout(
    toMountPoint(<AccelerationDetailsFlyout acceleration={acceleration} />)
  );
setRenderAccelerationDetailsFlyout(renderAccelerationDetailsFlyout);

return {
  renderAccelerationDetailsFlyout,
};
```